utils/src/test/java/org/opentripplanner/utils/color/ColorUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.utils.color;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.opentripplanner.utils.color.ColorUtils.computeBrightness;
+
+import java.awt.Color;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.FieldSource;
+
+public class ColorUtilsTest {
+
+  static final Arguments[] brightnessExpectations = {
+    arguments(Color.black, Brightness.DARK),
+    arguments(Color.green, Brightness.LIGHT),
+    arguments(Color.blue, Brightness.DARK),
+    arguments(Color.red, Brightness.LIGHT),
+    arguments(Color.yellow, Brightness.LIGHT),
+    arguments(Color.white, Brightness.LIGHT),
+    arguments(Color.pink, Brightness.LIGHT),
+    arguments(Color.orange, Brightness.LIGHT),
+    arguments(Color.cyan, Brightness.LIGHT),
+  };
+
+  @ParameterizedTest
+  @FieldSource("brightnessExpectations")
```

Comment:
While I'm in favour of it I know that using FieldSource will be controversial. Please use MethodSource if you want to avoid a long discussion.

Reply:
Just so you understand @miklcct, the fields(test-cases) will be sorted together according to the code-convetions, while test and test-cases should be together in the code for easy reading. It is also easier to construct test data in a method than in a constant. We usually use test-dta-genertors and these sometimes need an inisialized context (witch may not be inizilized when static fields are processed). 

Consider: In this case I do not see a problem other than if you add more tests to the same class. But for consistency, converting to a `@MethodSource` would be nice.